pymaker/approval.py

```diff
@@ -76,15 +76,17 @@ def hope_directly(**kwargs):
                  'name': 'can', 'outputs': [{'name': '', 'type': 'bool'}], 'payable': False, 'stateMutability': 'view',
                  'type': 'function'}]
 
-    def approval_function(token: ERC20Token, spender_address: Address, spender_name: str):
+    def approval_function(token: ERC20Token, spender_address: Address, spender_name: str, **kwargs):
```

Comment:
if I am not missing something, if moving **kwargs as param to approval_function then `def hope_directly(**kwargs):` should be changed to `def hope_directly():` and the `self.adapter.approve(hope_directly(from_address=usr), self.flipper.vat())` call within Collateral.approve should be changed to `self.adapter.approve(hope_directly(), source=self.flipper.vat(), from_address=usr)`

Reply:
In 09d8670881d1b803ab71641c3b126ab2e14eb7a6 I removed it from the `approval_function` to improve consistency with `directly`.